documents/policies/external/Competition Requirements.md

```diff
@@ -97,20 +98,38 @@ The WCA Delegate must submit the following information when requesting approval
+      3. If the registration requirements are not possible to be fulfilled by an individual due to reasons outside of their control, they are eligible for completing their registration following a suitable alternative approach, at the discretion of the Delegate. In case of such an inquiry, organizers must involve the Delegate, who must then carefully assess the individual's situation. If deemed appropriate, the Delegate and organizers must make reasonable efforts to provide an alternative approach for the individual to fulfilling the requirements.
+         1. Possible reasons include, but are not limited to:
+            1. Having an unsupported credit card for paying the registration fee.
+         2. Exceptions must not be based on:
+            1. Missing the registration deadline.
+            2. Not having enough money to pay the registration fee.
    2. The registration period(s) must be a clearly defined time range with a start and end time and date.
       1. Each registration period must be announced publicly on the WCA website at least 48 hours before it starts.
          1. The competition's WCA Delegate(s) and/or organizer(s) may register for the competition prior to the first registration period, however their registration(s) must not be approved until the first registration period has started.
       2. Different registration periods may have different registration requirements (e.g. restricting which events a competitor can register for).
-   3. Different locations for multi-location competitions may have different registration requirements (e.g. competitor limit).
+   3. The deadline for updating registered events for competitors with completed registrations must be outlined.
+      1. This deadline will either be at the same time as registration closes or after registration has closed.
+         1. The deadline may be set to be just before an event is scheduled to take place. 
+      2. Competitors may choose not to compete in an event they are registered for.
+   4. The organization team must outline the conditions of the waiting list, and if one is in place.
+      1. If a waiting list is in place:
+         1. Registrants will be placed on the waiting list if they have completed all requirements for a valid registration, but the competitor limit has been filled. Registrants from the waiting list will be accepted in the order that their registration was completed once a spot becomes available.
+         2. It must be stated what the deadline is to be accepted for a registrant on the waiting list.
+            1. This deadline should be the same as the deadline for refunds for competitors canceling their registration, if (partial) refunds are to be given.
+         3. The organization team may specify a limit to the waiting list. Once the limit of the waiting list has been reached, the Delegate should inform the WCAT to close the registration period.
```

Comment:
I like this. Does this limit have to be specified in advance, or will WCAT allow closing registrations once it gets to a high number, as they have in the past? Either way, it would be good to specify in the policy I feel.

Reply:
I agree with the thought behind this, but the implementation is awful. You should not require from delegates to manually notify WCAT at a certain point in time when this could just as well be managed by a software solution that automatically prevents registration once the limit is reached.